client/test/unit/specs/CobrsPhoneticMatchesRequest.spec.js

```diff
@@ -77,5 +77,9 @@ describe('store > checkManualCobrsPhoneticMatches', () => {
         expect(CobrsPhonetic.lastCall.args[0]).toEqual('/api/v1/requests/cobrsphonetics/dog  cat fish bear')
     })
 
+      it('escape special characters', ()=>{
+        store.dispatch('checkManualPhoneticMatches', '/?.><\\\'\":;\\|][}{=+_-)(*&^%$#@!~`ATHENAE/?.><\\\'\":;\\|][}{=+_-)(*&^%$#@!~`UM 139 LTD./?.><\\\'":;\\|][}{=+_-)(*&^%$#@!~`')
```

Comment:
This test fails - it should dispatch 'checkManualCobrsPhoneticMatches', not 'checkManualPhoneticMatches'. If you change that, this test then passes. 

Everything else is good. I will re-review and approve when you fix this.

Reply:
I fixed the name. Thanks.